modules/dataControllerModule/index.md

```diff
@@ -0,0 +1,30 @@
+# Overview
+
+```
+Module Name: Data Controller Module
+```
+
+# Description
+
+This module will filter EIDs and SDA based on the configurations.
+The filtered EIDs are stored in 'dcUsersAsEids' configuration and filtered SDA are updated in bidder configuration.
```

Comment:
Is this detail no longer relevant? I am not seeing any of this stored any longer, so if that is the case, can this be removed